test/SerialIOTest.cpp

```diff
@@ -91,8 +91,80 @@ TEST_CASE( "multi_series_test", "[serial]" )
     allSeries.clear();
 }
 
-inline
 void
+close_iteration_test( std::string file_ending )
+{
+    std::string name = "../samples/close_iterations_%T." + file_ending;
+
+    std::vector< int > data{ 2, 4, 6, 8 };
+    // { // we do *not* need these parentheses
+    #if 1
+    Series write( name, Access::CREATE );
+    {
+        Iteration it0 = write.iterations[ 0 ];
+        auto E_x = it0.meshes[ "E" ][ "x" ];
+        E_x.resetDataset( { Datatype::INT, { 2, 2 } } );
+        E_x.storeChunk( data, { 0, 0 }, { 2, 2 } );
+        it0.close( /* flush = */ false );
+    }
+    write.flush();
+    #endif
+    // }
+
+    /*
+     * This block will run fine if commented in
+     * But the following block will fail with a segfault
+     * upon adios_select_method ????
```

Comment:
that's because you cannot initialize ADIOS1 twice in the same process.
ADIOS1 uses some nasty static states.

Reply:
That explains things, was already suspecting something in this direction. So, the basic "close file" functionality should be working anyway for ADIOS1, but we should probably just run a simplified test, such as checking whether the file is on disk?

Reply:
Good idea, we can just use our helper methods to check we found the files on disk.